### PR TITLE
Move Marketplace link to profile dropdown

### LINKED ISSUE #451: Remove "Marketplace" nav item
I think its position in the top nav is way too prominent. I'd move it into the profile drop-down, underneath "Explore"

### PR DESCRIPTION
**Pull request by pablopunk**
screen shot 2017-07-04 at 6 51 15 pm

Closes #451

### CODE REVIEW

**Comment by bfred-it** [comment]
Excellent!

File: src/content.js [hunk 1]
```diff
@@ -514,6 +523,8 @@ async function onDomReady() {
 
 	markUnread.setup();
 
+	addMarketplaceToProfileDropDown();
+
```
Comment (bfred-it) [inline src/content.js]: You're already using `elementReady`, so you can move this out of `onDomReady` and in `init`

File: src/content.js [hunk 2]
```diff
@@ -114,6 +114,12 @@ function hideEmptyMeta() {
 	}
 }
 
+function addMarketplaceToProfileDropDown() {
```
Comment (sindresorhus) [inline src/content.js]: `addMarketplaceToProfileDropDown` => `moveMarketplaceLinkToProfileDropdown`

**Comment by sindresorhus** [comment]
Awesome. Thank you @pablopunk :)

**Pull request merged**